sudoers.d/ceph-osd-smartctl

```diff
@@ -0,0 +1,3 @@
+## allow ceph-osd (which runs as user ceph) to collect device health metrics
+
+ceph ALL=NOPASSWD: /usr/sbin/smartctl -a --json /dev/*
```

Comment:
I'm not sure about the need of specifying the arguments.
I tried locally and this syntax doesn't prevent from running any other smartctl options.

Reply:
Can you retest?  On my system (fedora 28) I verified that the given arguments were required and that the /dev/* didn't match e.g. /dev/*/*.  I'm not sure I verified new arguments couldn't be added at the end, though, but I don't think smartctl can do anything dangerous there.

Reply:
You are perfectly true. I add another sudoers overriding that one :(